### ISSUE TITLE
Graphics tablet input events aren't received by the-machinery

### ISSUE THREAD

**Issue by photex** [OurMachinery/themachinery-public#267]
Howdy!

I read in another issue that this is likely due to using raw input. But I wanted to report it anyway so you could put in a pin in it for later perhaps.
Tablet input is fairly common I think, even if it's just used like a mouse, and I sometimes use it as an alternative to a mouse just to alleviate wrist strain. :D

I use an XP-Pen Star G640 with whatever default Windows drivers were found for it. (Windows 10)

Cheers!
Chip

**Comment by niklas-ourmachinery** [comment]
I agree, we should support tablet input everywhere in the UI. We support the Surface Book Pen in the Simple Draw executable, but I need to do some more testing to see what else works and not. (The cats ate my pen, so I can't test it right now -- need to order a new one 😄 ).
